tensorflow/lite/micro/kernels/arc_mli/mli_slicers.cc

```diff
@@ -0,0 +1,129 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "mli_slicers.h"
+
+#define MAX(A,B) (((A) > (B))? (A): (B))
```

Comment:
Can we use std::max/min here and below?

Reply:
Sure, why not! I replaced it here and in 2 more related files with the same macro. Plese check.